lib/http.js

```diff
@@ -496,10 +518,12 @@ async function _verifyDelegation({
 
     return defaultDocumentLoader(url);
   });
-
+  const expectedRootCapability = _getExpectedRootFromInvocationTarget(
+    {target: invocationTarget, keystoreId});
```

Comment:
can we just use `invocationTarget` throughout instead of renaming to `target`?

Reply:
good idea and done here: https://github.com/digitalbazaar/bedrock-kms-http/pull/14/commits/557d6a651808df897c73cc6ce3055dd21f35835f

The other PR in bedrock-edv-storage should use it too.